manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -2286,10 +2286,6 @@ metadata:
   name: gatekeeper-webhook-service
   namespace: gatekeeper-system
 spec:
-  ports:
```

Comment:
It looks like the change removed this from the static manifest.

Instead of removing the ports from the `config/` directory, can we alter the kustomization under the `cmd/helmify` directory to remove the ports?

Here is an example that does it for the Deployment:


https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/delete-ports.yaml